casepro/statistics/migrations/0008_existing_case_stats_count.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+def calculate_totals_for_cases(apps, schema_editor):
+    from casepro.cases.models import Case, CaseAction
```

Comment:
We want to avoid referencing actual model classes here so that future changes don't break old migrations, and use `apps.get_model(..)` instead. Instead of calling `DailyCount.record_item`, manually create the objects etc.

If you look at casepro/statistics/migrations/0002_populate_reply_counts.py you can see how this was done last time we added a data migration to backfill daily counts.